### PR TITLE
Add lostintangent.workspace-layout extension

### PR DESCRIPTION
**Pull request by sevetseh28**
- [x] I have read the note above about PRs contributing or fixing extensions
- [x] I have tried reaching out to the extension maintainers about publishing this extension to Open VSX (if not, please create an issue in the extension's repo using [this template](https://github.com/open-vsx/publish-extensions/blob/HEAD/docs/external_contribution_request.md)).
- [x] This extension has an [OSI-approved OSS license](https://opensource.org/licenses) (we don't accept proprietary extensions in this repository)

## Description

This PR adds extension `lostintangent.workspace-layout` to the OpenVSX marketplace.

### CODE REVIEW

**Comment by filiptronicek** [comment]
Linking the upstream issue: https://github.com/lostintangent/workspace-layout/issues/10

**Comment by sevetseh28** [comment]
Thanks @filiptronicek , do we need to wait for the maintainer's approval or could we directly merge it?

Thanks!

**Comment by filiptronicek** [comment]
@sevetseh28 we do not since the extension is MIT-licensed (but of course always nice to ask 😉). 

It seems that the extension is building fine, which means it should be good to merge. Thanks for your contribution!

**Review by filiptronicek — APPROVED** [review]

**Pull request merged**
